crates/astria-sequencer-relayer/src/relayer/mod.rs

```diff
@@ -82,6 +84,9 @@ pub(crate) struct Relayer {
     // If this is set, only relay blocks to DA which are proposed by the same validator key.
     validator: Option<Validator>,
 
+    // The rollup IDs to include in submissions (all rollups if filter is empty).
+    rollup_id_filter: HashSet<RollupId>,
```

Comment:
Could this be made a `Arc<HashSet<RollupId>>` since this will never be mutated again? This way we can avoid the (moderatly expensive) clones.

Reply:
Done in [02b1276](https://github.com/astriaorg/astria/pull/1001/commits/02b1276267c6ac65b72c78191ebecdfae92e72d9).